README.md

```diff
@@ -97,7 +104,7 @@ This function splits the contents of the target column on the `char` (e.g, `|`)
 
 ### lookup
 
-Usage: `lookup(table, column, column2)`
+Usage: `lookup("table", "column", "column2")`
```

Comment:
Now it seems weird to me that `table.column` is fine, but here we require `"table"` and `"column"`.

Reply:
With restricting datatypes to a single, unquoted word, unless you surround the table/column in double quotes, it thinks it's a datatype.

I guess the easy solution is to go back to using `table.column` for all of these, even though the table has to be the same?

Reply:
Now I think we're making a mistake by building too much semantics into the grammar, when we really just want to know the shapes of things. I'll think about this some more.

Reply:
See https://github.com/ontodev/valve.js/pull/11

Reply:
Updated the docs for the simplified grammar